docs/data/trigger-sync.md

```diff
@@ -0,0 +1,270 @@
+If you need to trigger sync based on a different condition, you need to create your own module and adjust the module logic accordingly.
+Additional examples are available in this [GitHub Repo](https://github.com/viam-labs/trigger-sync-examples-v2).
+
+To use [`sync-at-time:timesyncsensor`](https://app.viam.com/module/naomi/sync-at-time):
+
+1. Go to your machine's **Config** page and click **Add Component**.
+2. Then select the `sync-at-time:timesyncsensor` model from the [`sync-at-time` module](https://app.viam.com/module/naomi/sync-at-time).
+3. Click **Add module**, then enter a name for your sensor and click **Create**.
+   The sensor will return true and enable sync when in a specified time frame.
+4. To configure your timeframe, go to the new component panel and copy and paste the following attribute template into your sensor’s **Attributes** box:
+
+   {{< tabs >}}
+   {{% tab name="Config builder" %}}
+
+```json
+{
+  "start": "HH:MM:SS",
+  "end": "HH:MM:SS",
+  "zone": "<TIMEZONE>"
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json
+{
+  "name": "<SENSORNAME>",
+  "model": "naomi:sync-at-time:timesyncsensor",
+  "type": "sensor",
+  "namespace": "rdk",
+  "attributes": {
+    "start": "HH:MM:SS",
+    "end": "HH:MM:SS",
+    "zone": "<TIMEZONE>"
+  },
+  "depends_on": []
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json
+{
+  "name": "timesensor",
+  "model": "naomi:sync-at-time:timesyncsensor",
+  "type": "sensor",
+  "namespace": "rdk",
+  "attributes": {
+    "start": "18:29:00",
+    "end": "18:30:00",
+    "zone": "CET"
+  },
+  "depends_on": []
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The following attributes are available for the `naomi:sync-at-time:timesyncsensor` sensor:
+
+<!-- prettier-ignore -->
+| Name    | Type   | Inclusion    | Description |
+| ------- | ------ | ------------ | ----------- |
+| `start` | string | **Required** | The start time for the time frame during which you want to sync, example: `"14:10:00"`.  |
+| `end`   | string | **Required** | The start time for the time frame during which you want to sync, example: `"15:35:00"`. |
+| `zone`  | string | **Required** | The time zone for the `start` and `end` time, for example: `"CET"`. |
+
+In the next step you will configure the data manager to take the sensor into account when syncing.
+
+### Configure data manager to sync based on sensor
+
+On your machine's **Config** tab, switch to **JSON** mode and add a `selective_syncer_name` with the name for the sensor you configured:
+
+{{< tabs >}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers" data-line="6"}
+{
+  "attributes": {
+    "additional_sync_paths": [],
+    "capture_dir": "",
+    "capture_disabled": false,
+    "selective_syncer_name": "<SENSOR-NAME>",
+    "sync_disabled": false,
+    "sync_interval_mins": 0.1,
+    "tags": []
+  },
+  "name": "Data-Management-Service",
+  "type": "data_manager"
```

Comment:
```suggestion
  "name": "Data-Management-Service",
  "type": "data_manager",
  "attributes": {
    "additional_sync_paths": [],
    "capture_dir": "",
    "capture_disabled": false,
    "selective_syncer_name": "<SENSOR-NAME>",
    "sync_disabled": false,
    "sync_interval_mins": 0.1,
    "tags": []
  }
```
[opt] This is the order in which it shows up on https://docs.viam.com/data/capture/#add-the-data-management-service so people can recognize it easily,  and is more intuitive